examples/library-example/library-example.cabal

```diff
@@ -12,7 +12,7 @@ category:            Games
 build-type:          Simple
 extra-doc-files:     README.md
                    , CHANGELOG.md
-tested-with:         GHC == 8.6.5
+tested-with:         GHC == 8.8.3
```

Comment:
this file (and directory) and directory has been deleted. the library example lives now in another repo so that it can be easily cloned, see https://github.com/hasktorch/hasktorch-skeleton.